README.md

```diff
@@ -43,8 +43,10 @@ It supports the following formats:
 It provides 3 C++ classes that are the starting point of every morphology analysis:
 - `Soma`: contains the information related to the soma.
 
-- `Section`: a section is the succession of points between two bifurcations. To the bare minimum
-the `Section` object will contain the section type, the position and diameter of each point.
+- `Section`:
+  - A succession of points between two bifurcations if the file is in SWC format.
+  - A succession of points (not necessarily between two bifurcations) in an S-exp if the file is in ASC format.
```

Comment:
what is S-exp?

Reply:
A Lisp thing; the neurolucida format reminded me of it initially:

https://en.wikipedia.org/wiki/S-expression

Reply:
I see, very geeky!